compiler/rustc_parse/src/parser/attr.rs

```diff
@@ -25,6 +25,7 @@ pub enum InnerAttrForbiddenReason {
     AfterOuterAttribute { prev_outer_attr_sp: Span },
 }
 
+#[derive(PartialEq)]
```

Comment:
Where is this used?

Reply:
was left unintentionally.
I removed it now.